logparser/logs/micro/error_logs.json

```diff
@@ -0,0 +1 @@
rces exceeded."}, "12": {"url": "https://community.rti.com/static/documentation/connext-micro/2.4.9/doc/html/group__DBLogCodesClass.html#ga8715519eb384ce72f7f73a4118c7db91", "name": "DB_LOG_OUT_OF_RECORDS_EC", "description": "Table record resources exceeded."}, "15": {"url": "https://community.rti.com/static/documentation/connext-micro/2.4.9/doc/html/group__DBLogCodesClass.html#gac9ba1cc497002c788c8e8f00456dc39a", "name": "DB_LOG_RECORDS_INUSE_EC", "description": "Cannot delete table, records are still in table."}, "14": {"url": "https://community.rti.com/static/documentation/connext-micro/2.4.9/doc/html/group__DBLogCodesClass.html#ga55f58ba4c0617d1af47873bb59af760c", "name": "DB_LOG_OUT_OF_CURSORS_EC", "description": "Table cursor resources exceeded."}, "17": {"url": "https://community.rti.com/static/documentation/connext-micro/2.4.9/doc/html/group__DBLogCodesClass.html#ga0cc33bcd6df1d550d24ebff5b8eca0d6", "name": "DB_LOG_INDEX_INUSE_EC", "description": "Cannot delete table, indices are still in use."}, "16": {"url": "https://community.rti.com/static/documentation/connext-micro/2.4.9/doc/html/group__DBLogCodesClass.html#gaa8f12ab9d62b3e8652ec2421b0a3a011", "name": "DB_LOG_CURSORS_INUSE_EC", "description": "Cannot delete table, cursors are still in use."}, "19": {"url": "https://community.rti.com/static/documentation/connext-micro/2.4.9/doc/html/group__DBLogCodesClass.html#gaa2368e81eeb4e65b9d31d9c51ea4f8cc", "name": "DB_LOG_ALLOC_INDEX_POOL_EC", "description": "Failed to allocate index buffer pool."}, "21": {"url": "https://community.rti.com/static/documentation/connext-micro/2.4.9/doc/html/group__DBLogCodesClass.html#ga8a940a53e5697d3ae80f70d17137ddcb", "name": "DB_LOG_ALLOC_DATABASE_EC", "description": "Failed to allocate database."}, "22": {"url": "https://community.rti.com/static/documentation/connext-micro/2.4.9/doc/html/group__DBLogCodesClass.html#gab0b08400071739477ddf8032b78561bb", "name": "DB_LOG_TABLE_NOT_INUSE_EC", "description": "An attempt was made to delete a table not in use."}, "23": {"url": "https://community.rti.com/static/documentation/connext-micro/2.4.9/doc/html/group__DBLogCodesClass.html#ga77bd8c60e6d6941ec99bb58310c8ca5d", "name": "DB_LOG_RECORD_ALREADY_EXISTS_EC", "description": "An attempt was made to insert an already existing record."}, "24": {"url": "https://community.rti.com/static/documentation/connext-micro/2.4.9/doc/html/group__DBLogCodesClass.html#ga25f8d68ce8664b38e22b8ad116ea54f2", "name": "DB_LOG_RECORD_DOES_NOT_EXIST_EC", "description": "An attempt was made to delete a record that does not exist."}, "25": {"url": "https://community.rti.com/static/documentation/connext-micro/2.4.9/doc/html/group__DBLogCodesClass.html#ga38239bb01e26316d5726bf1394baa9d4", "name": "DB_LOG_CURSOR_INVALIDATED_EC", "description": "An attempt was made to use a cursor that has been invalidated."}, "26": {"url": "https://community.rti.com/static/documentation/connext-micro/2.4.9/doc/html/group__DBLogCodesClass.html#ga8cc42b8ad1be9aff1
```

Comment:
Have you tried with the standalone .zip file? Is it able to find the .json file inside it? If it is not possible to have the json inside the .zip file we will need to update the README with instructions to parse the micro logs.